docs/content/quickstart.md

```diff
@@ -22,22 +22,53 @@ with standard OCP and Kubernetes toolchains.
 
 ## Create a cluster
 
+In the following instructions, shell variables are used to indicate values that you should adjust to your own environment.
+
 1. Install the HyperShift CLI using Go 1.16+:
 
         go install github.com/openshift/hypershift@latest
 
-1. Install HyperShift into the management cluster:
+1. Install HyperShift into the management cluster
+
+    If running your cluster on AWS, create a public s3 bucket to hold OIDC discovery documents of guest clusters:
```

Comment:
Can we please mention the s3 bucket creation in the ## Prerequisites section?
Can we please also elaborate in this section on how to create the other prerequisite `* A Route53 public zone for the cluster's DNS records`